rootfs/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.5
+RUN apt-get upgrade && apt-get install -y && curl -sSL https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py | python - && pip install flake8 
+WORKDIR /tmp
```

Comment:
This line feels odd to me.  Is this because there will be more lines that follow where temporary files will be downloaded perhaps?  Maybe for building some additional tools from source?  If so, I get where you're going, but...

Generally, I would think that `/tmp` is not the ideal working directory for someone who would be using this image.  IMO, people using this image should mount their python code anywhere they wish in the container (i.e. using `-v <local path>:<container path>`) and then they can select the working directory themselves (i.e. using `-w <container path>`)